js/ui/select_box.d.ts

```diff
@@ -91,13 +91,13 @@ export type OpenedEvent = EventInfo<dxSelectBox>;
 export type OptionChangedEvent = EventInfo<dxSelectBox> & ChangedOptionInfo;
 
 /** @public */
-export type PasteEvent = NativeEventInfo<dxSelectBox>;
+export type PasteEvent = NativeEventInfo<dxSelectBox, ClipboardEvent>;
 
 /** @public */
 export type SelectionChangedEvent = EventInfo<dxSelectBox> & SelectionChangedInfo;
 
 /** @public */
-export type ValueChangedEvent = NativeEventInfo<dxSelectBox> & ValueChangedInfo;
+export type ValueChangedEvent = NativeEventInfo<dxSelectBox, KeyboardEvent | MouseEvent | UIEvent> & ValueChangedInfo;
```

Comment:
```suggestion
export type ValueChangedEvent = NativeEventInfo<dxSelectBox, KeyboardEvent | MouseEvent | Event> & ValueChangedInfo;
```
https://github.com/DevExpress/DevExtreme/pull/19950/files#r741000230